docs/source/yelpsoa_configs.rst

```diff
@@ -843,15 +593,38 @@ Each instance MAY have:
 
   * Anything in the `Common Settings`_.
 
-  * ``net``
+  * ``net``: Specify which kind of
+    `networking mode <https://docs.docker.com/engine/reference/run/#network-settings>`_
+    instances of this service should be launched using. Defaults to ``'bridge'``.
 
-  * ``cmd``
+  * ``cmd``: The command that is executed. Can be used as an alternative to
+    args for containers without an `entrypoint
+    <https://docs.docker.com/reference/builder/#entrypoint>`_. [#note]_.
 
-  * ``args``
+  * ``args``: An array of docker args if you use the `"entrypoint"
+    <https://docs.docker.com/reference/builder/#entrypoint>`_ functionality. [#note]_.
 
-  * ``deploy_group``
+  * ``deploy_group``: A string identifying what deploy group this instance belongs
+    to. The ``step`` parameter in ``deploy.yaml`` references this value
+    to determine the order in which to build & deploy deploy groups. Defaults to
+    ``clustername.instancename``. See the deploy group doc_ for more information.
```

Comment:
hmm, are these not in the kubernetes settings? imo, we should define these there and update the old `See the `marathon-[clustername].yaml`_ section for details for each of these parameters.` text to refer to the `eks/kubernetes-[clustername].yaml` files

Reply:
I removed the marathon-clustername.yaml section since we dont have these files in soaconfigs anymore and added them under kubernetes-clustername.yaml since theyre applicable there